nbs/01_io.ipynb

```diff
@@ -608,9 +608,9 @@
     "    \n",
     "    # Perform transformation and filtering\n",
     "    globs = {'pd':pd, 'np':np, 'stk':stk, 'df':df, **constants}\n",
-    "    if 'preprocessing' in desc:  exec(desc['preprocessing'], globs)\n",
-    "    if 'filter' in desc: globs['df'] = globs['df'][eval(desc['filter'], globs)]\n",
-    "    if 'postprocessing' in desc and not skip_postprocessing: exec(desc['postprocessing'],globs)\n",
+    "    if 'preprocessing' in desc and desc['preprocessing']: exec(desc['preprocessing'], globs)\n",
```

Comment:
@rrebane the shorter version of the same thing is
`if desc.get('preprocessing'):`

Reply:
And the same applies to other three lines

Reply:
Good point. Changed it.